graphbolt/include/utils.h

```diff
@@ -0,0 +1,151 @@
+/**
+ *  Copyright (c) 2023 by Contributors
+ * @file graphbolt/include/csc_sampling_graph.h
```

Comment:
Filename did not match. `utils.h` is perhaps to generic. How about `serialize.h`?